Add an example configuration for metallb.universe.tf/allow-shared-ip (#1124)
metallb/metallb: website/content/usage/_index.md

work around a Kubernetes limitation, and to work with limited IP
addresses.

> Example configuration for two services you want to share the same ip address.

Reviewer: ```suggestion
Here is an example configuration of two services that share the same ip address:
```
